src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -693,7 +693,7 @@
     <value>JsonPropertyInfo '{0}' defined in type '{1}' is marked both as required and as an extension data property. This combination is not supported.</value>
   </data>
   <data name="JsonRequiredPropertiesMissing" xml:space="preserve">
-    <value>JSON deserialization for type '{0}' was missing required properties, including the following: {1}</value>
+    <value>JSON deserialization for type '{0}' was missing required properties including: {1}.</value>
```

Comment:
consider adding quotes around `{1}` seems we do that elsewhere

Reply:
The `{1}` is substituted with comma separated values of property names which are quoted by the throw helper.